docs/migration/fastapi.rst

```diff
@@ -174,8 +174,290 @@ and to easily access dependencies from higher levels.
 
     * :doc:`/usage/dependency-injection`
 
+Lifespan
+~~~~~~~~
+
+If you're using an async context manager and pass parameters to it, most likely the order of parameters is inversed between FastAPI and Litestar.
+
+.. tab-set::
+
+    .. tab-item:: FastAPI
+        :sync: fastapi
+
+        .. code-block:: python
+
+            @asynccontextmanager
+            async def lifespan(
+                _app: FastAPI,
+                app_settings: AppSettings,
+            ):
+                # Setup code here
+                yield
+                # Teardown code here
+
+    .. tab-item:: Litestar
+        :sync: litestar
+
+        .. code-block:: python
+
+            @asynccontextmanager
+            async def lifespan(
+                app_settings: AppSettings,
+                _app: Litestar,
+            ):
+                # Setup code here
+                yield
+                # Teardown code here
+
+~~~~~~~
+Cookies
+~~~~~~~
+
+While with FastAPI you usually set cookies on the response `Response` object, in Litestar it will be set at the decorator level.
+
+.. tab-set::
+
+    .. tab-item:: FastAPI
+        :sync: fastapi
+
+        .. code-block:: python
+
+            @app.get("/")
+            async def index(response: Response) -> dict[str, str]:
+                response.set_cookie(key="my_cookie", value="cookie_value")
+                ...
+
+    .. tab-item:: Litestar
+        :sync: litestar
+
+        .. code-block:: python
+
+            @get(response_cookies={"my-cookie": "cookie-value"})
+            async def handler() -> str:
+                ...
+
+
+Dependencies parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+The way dependencies parameters are passed differs between FastAPI and Litestar, note the `state: State` parameter in the Litestar example.
```

Comment:
maybe point out the difference between `state`, `request.state` and `request.app.state`?

Reply:
well i wrote something, I find it confusing, so im hapy to a better rewording